src/types.d.ts

```diff
@@ -96,6 +98,8 @@ declare global {
     // Extra YAML section entries that are stringd and serve
     // to provide additional metadata about the API doc entry
     tags: Array<string>;
+    // The raw file content
+    rawContent: string;
```

Comment:
Not sure if this will accidentally bloat the whole thing. I'm against storing two copies of the same thing. The AST tree should have all you need.

Reply:
The issue is, things like `{Array}` are transformed into `` `<Array>` ``, and the "raw" content isn't the same.

Reply:
But does that matter, and if yes, why? Afaik these are already converted to Markdown Links on the AST, so you could simply make them HTML links?

Reply:
It matters because it's not 1:1 with the original. (But TBH i'd rather have it not exactly 1:1 than have this much bloat)